include/nonius/param.h++

```diff
@@ -0,0 +1,176 @@
+// Nonius - C++ benchmarking tool
+//
+// Written in 2014 by Martinho Fernandes <martinho.fernandes@gmail.com>
```

Comment:
Hehe, yup! 

I think long term it's easiest to put something like _"by the nonius contributors nonius@rmf.io, see AUTHORS file."_ Then the `AUTHORS` file may list maintainers, contributors, etc.